extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -103,6 +103,9 @@ export class AmpIframe extends AMP.BaseElement {
     /** @private {string} */
     this.sandbox_ = '';
 
+    /** @private {Function} */
+    this.unlisten_ = null;
```

Comment:
Let's name it `unlistenPym_`.

Reply:
Done in 0b31ccaa5d88c6b2d8498554fbb046a8f9ddc2e9